docs/glossary.md

```diff
@@ -0,0 +1,171 @@
+directives. Also See **Subschema**.
+
+### Apollo Graph Platform
+
+Branding used by Apollo for describing their open-source products.
+
+> The leading open-source GraphQL implementation with 17M downloads per month
+> and the only end-to-end cloud management solution for GraphQL. Source:
+> [https://www.apollographql.com/](https://www.apollographql.com/) (2022/02/24)
+
+### Apollo Studio
+
+A schema registry for registering and validating Apollo federation subgraphs.
+
+### Data Source
+
+A “dumb” service or database that exposes raw data. Often a data source is
+consumed by a service, for reading and/or persisting business logic state.
+
+It is hard to define a border between Data Source and Service. E.g. a PostgreSQL
+database could be considered a data source. If the PostgreSQL database uses
+database triggers for applying business logic constraints it could be considered
+a **Service** instead. The boundaries are fluid.
+
+# Global Object Identification
+
+Pattern of having global unique IDs between all Object Types (entities), usually
+on a field with the name `id` and the field type `ID!`.
+
+- [Global Object Identification (graphql.org)](https://graphql.org/learn/global-object-identification/)
+- [GraphQL Global Object Identification Specification (relay.dev)](https://relay.dev/graphql/objectidentification.htm)
+
+### GraphQL Hive
+
+A GraphQL schema registry and workflow tool maintained by The Guild that is
+compatible with any **type of merging** workflow.
+
+### GraphQL Mesh
+
+A library maintained by The Guild that is capable of transforming a single or
+many **data sources** and **services** (GraphQL API, REST API, SQL Database)
+into a GraphQL schema. In the case of multiple data sources or services, Mesh
+uses Schema Stitching for doing Schema Composition of the GraphQL subschemas
+that were the results of the transform.
+
+### Hybrid Schema Composition
+
+A mix of both **Subschema Directive-based Schema Composition** and
+**Programmatic/Config Gateway-based Schema Composition**.
+
+### Nadel ([Link](https://github.com/atlassian-labs/nadel))
+
+> Nadel is a Kotlin library to combine several GraphQL services together. This
+> is achieved by combining several underlying GraphQL services (schemas) into
+> one schema (overall schema). The main purpose of this library is to power a
+> GraphQL gateway which serves as the entry point for executing requests across
+> different services while presenting one GraphQL API.
+
+### Nautilus ([Link](https://github.com/nautilus/gateway))
+
+> A library and standalone service that composes your GraphQL APIs into one
+> endpoint.
+
+Uses these
+[merge strategies](https://github.com/nautilus/gateway/blob/master/docs/mergingStrategies.md)
+and one interface type to compose subschemas (see **Node interface**).
+
+### Node interface
+
+The [Global Object
+Identification][https://graphql.org/learn/global-object-identification/]
+[spec][https://relay.dev/graphql/objectidentification.htm] defines a `Node`
```

Comment:
Links fixes
```suggestion
The [Global Object
Identification](https://graphql.org/learn/global-object-identification/)
[spec](https://relay.dev/graphql/objectidentification.htm) defines a `Node`
```

Alternatively, you can use the `[words][link-ref-name]` followed by `[link-ref-name]: https://...` syntax like below 🤷